README.md

```diff
@@ -12,6 +12,15 @@ https://chrome.google.com/webstore/detail/confluence-better-pagetre/impgjbnbnoch
 
 https://user-images.githubusercontent.com/1780339/136763140-d18000d3-b736-468b-b6a7-c4b663e9e71d.mov
 
+## Keyboard shortcuts
+
+| keys | action |
+|-----|-----|
+| Cmd + Shift + K | Enable Extension |
+| Cmd + K | Search |
```

Comment:
```suggestion
| Cmd + K (Ctrl + space) | Search |
```

For windows user, <kbd>ctrl</kbd>+<kbd>space</kbd>. Please see the document.
https://fablic.qiita.com/kissy0601/items/ab4227a1141d9ae294fe